src/python_bindings/gfd/bind_gfd_verification.cpp

```diff
@@ -0,0 +1,34 @@
+#include "bind_gfd_verification.h"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "algorithms/gfd/mining_algorithms.h"
+#include "py_util/bind_primitive.h"
+
+namespace {
+namespace py = pybind11;
+}  // namespace
+
+namespace python_bindings {
+void BindGfdVerification(pybind11::module_& main_module) {
+    using namespace algos;
+
+    auto gfd_module = main_module.def_submodule("gfd_verification");
+
+    py::class_<graph_t>(gfd_module, "Graph_t");
+
+    py::class_<Gfd>(gfd_module, "Gfd")
+            .def(py::init<graph_t&, std::vector<Literal>&, std::vector<Literal>&>())
+            .def("GetPattern", &Gfd::GetPattern)
+            .def("GetPremises", &Gfd::GetPremises)
+            .def("GetConclusion", &Gfd::GetConclusion)
+            .def("SetPattern", &Gfd::SetPattern)
+            .def("SetPremises", &Gfd::SetPremises)
+            .def("SetConclusion", &Gfd::SetConclusion);
+
+    BindPrimitive<GfdValidation, EGfdValidation, NaiveGfdValidation>(
+            gfd_module, &GfdHandler::GfdList, "GfdAlgorithm", "get_gfds",
+            {"GfdValid", "EGfdValid", "NaiveGfdValid"});
```

Comment:
```suggestion
            {"GfdValid", "EGfdValid", "NaiveGfdValid"}, py::return_value_policy::copy);
```
There is actually an error in the bindings and the core library/pybind11 related to this part right now. Looks like it is not relevant to GFDs, and passing the `copy` policy instead of the default (mostly broken) policy will work fine.